mainApp/src/main/java/logisticsmarshall/tqs/ua/services/DeliveryService.java

```diff
@@ -102,8 +102,12 @@ private Driver validateDeliveryChange(User user, Delivery delivery, boolean shou
         Driver driver = user.getDriver();
         if (driver == null
                 || driver.getPhoneNo().isEmpty()
-                || Boolean.FALSE.equals(driver.getStatus()))
+                || Boolean.TRUE.equals(driver.getStatus()))
```

Comment:
this is a lowercase boolean now, please simplify

Reply:
done